tests/PHPStan/Analyser/data/integer-range-types.php

```diff
@@ -232,8 +232,8 @@ public function math($i, $j, $z, $pi, $r1, $r2, $rMin, $rMax, $x, $y) {
 		assertType('int<-2, 9>|int<21, 30>', $r1 - $z);
 		assertType('int<-200, -20>|int<1, 30>', $r1 * $z);
 		assertType('float|int<0, 10>', $r1 / $z);
-		assertType('int<min, 15>', $rMin * $z);
-		assertType('int<-100, max>', $rMax * $z);
+		assertType('int', $rMin * $z);
```

Comment:
Multiplication by $z leads to a integer range with no upper bound unioned with a range without a lower bound.